src/Marten/Events/EventGraph.cs

```diff
@@ -130,6 +129,69 @@ public void MapEventType(Type eventType, string eventTypeName)
             eventMapping.EventTypeName = eventTypeName;
         }
 
+        public IEventStoreOptions Upcast<TEvent>(
```

Comment:
@jeremydmiller, maybe it would be worth grouping those transformations into nested property instead of adding them straight into `EventGraph`? E.g. `options.Events.Transformations.Upcast(...)`. Thoughts?

Reply:
Maybe, but that's a lot of dots and not super discoverable. I think I'd leave it on `EventGraph`, but maybe look at what could be hidden or simplified in v6.